examples/rgw-cache/nginx.conf

```diff
@@ -0,0 +1,40 @@
+
+user  nginx;
+#Process per core
+worker_processes  auto;
+pid        /var/run/nginx.pid;
+
+events {
+#Number of connections per worker
+    worker_connections  1024;
+}
+
+
+http {
+	lua_package_path '/usr/local/openresty/lualib/?.lua;;'; 
+	aws_auth $aws_token {
+    # access key and secret key of the cache 
+        access_key cache;
```

Comment:
should declare that the user should change it with the one of the user with amz-cache caps